spec/integration/shared-examples.js

```diff
@@ -162,6 +166,22 @@ export default function IntegrationTestsForGateway (gateway, params, extraTests)
     })
   })
 
+  fdescribe('with timeout and enableHTTP408OnTimeouts=true', () => {
```

Comment:
Small detail here (`fdescribe` instead of `describe`)

Reply:
oopsie! 😊